core/trino-main/src/main/java/io/trino/execution/StageStateMachine.java

```diff
@@ -383,10 +383,14 @@ public StageInfo getStageInfo(Supplier<Iterable<TaskInfo>> taskInfosSupplier)
         long internalNetworkInputPositions = 0;
 
         long rawInputDataSize = 0;
+        long rawNonPartitionedInputDataSize = 0;
```

Comment:
We already have `physicalInputDataSize`. How it's different?

Reply:
Didn't realize that `physicalInputDataSize` was introduced for this purpose. After checking the implementation of `physicalInputDataSize`, and I agree that it fits to this purpose and is implemented in a right way.  
Let me close this PR.